test/health_check_test_case.py

```diff
@@ -118,7 +115,7 @@ def test_health_endpoint_keys(self):
         self._create_mock_queues()
         for keys, expected_response in expected.items():
             with self.subTest(msg=keys):
-                with ResponsesHelper() as helper:
+                with responses.RequestsMock() as helper:
```

Comment:
The commit that touches this line should remove the ResponseHelper and deal with AuthResponseHelper. Or create FIXME+ticket to do that.

Reply:
ill merge the commit that attics the retorts file with this commit.